src/Utilities/FNADllMap.cs

```diff
@@ -0,0 +1,184 @@
+#region License
+/* FNA - XNA4 Reimplementation for Desktop Platforms
+ * Copyright 2009-2021 Ethan Lee and the MonoGame Team
+ *
+ * Released under the Microsoft Public License.
+ * See LICENSE for details.
+ */
+#endregion
+
+#if NET5_0
```

Comment:
I would suggest to use the `NET5_0_OR_GREATER` notation as described here: https://github.com/dotnet/designs/blob/main/accepted/2020/or-greater-defines/or-greater-defines.md

Reply:
It looks like those defines are only available in .NET 6+, but there is a generic `NET` define that means 5.0+. We'll use that one instead.